src/test/java/marquez/db/mappers/JobRowMapperTest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.db.mappers;
+
+import static marquez.common.models.CommonModelGenerator.newDatasetUrn;
+import static marquez.common.models.CommonModelGenerator.newDescription;
+import static marquez.common.models.CommonModelGenerator.newJobName;
+import static marquez.common.models.CommonModelGenerator.newLocation;
+import static marquez.common.models.Description.NO_DESCRIPTION;
+import static marquez.db.models.DbModelGenerator.newNamespaceRow;
+import static marquez.db.models.DbModelGenerator.newRowUuid;
+import static marquez.db.models.DbModelGenerator.newTimestamp;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.net.URI;
+import java.sql.Array;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.UUID;
+import marquez.UnitTests;
+import marquez.common.models.Description;
+import marquez.common.models.JobName;
+import marquez.db.Columns;
+import marquez.service.models.Job;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+@Category(UnitTests.class)
+public class JobRowMapperTest {
+  private static final UUID ROW_UUID = newRowUuid();
+  private static final JobName NAME = newJobName();
+  private static final URI LOCATION = newLocation();
+  private static final UUID NAMESPACE_UUID = newNamespaceRow().getUuid();
+  private static final Description DESCRIPTION = newDescription();
+  private static final String[] INPUT_DATASET_URNS = new String[] {newDatasetUrn().toString()};
```

Comment:
`newDatasetUrn().toString()` returns:

```
 Urn(value=urn:dataset:analytics_db:public.room_bookings_7_days)
```

You'll want to use `newDatasetUrn().getValue()` instead:

```
 urn:dataset:analytics_db:public.room_bookings_7_days
```

Reply:
Updated